doc/adrs/158_add_undo_redo_support.adoc

```diff
@@ -0,0 +1,127 @@
+= ADR-158 Add undo redo support
+
+== Context
+
+Undo redo is not implemented on representations.
```

Comment:
This ADR only mentions representations and purely frontend state.
The real state of a project is on the backend (even in the DB), representations are only partial, ambiguous and opaque projections of that state.

I can currently define a tool which applies to a node on a diagram, makes changes on the underlying semantic data on the backend, but produces zero visible change on the frontend. A purely frontend approach to undo/redo can not handle this.

Reply:
After more discussions, I completly changed the approach